bin/package_build.py

```diff
@@ -15,45 +15,10 @@ def purify(dirty):
 	cleaner = clean.replace('"', '')
 	return cleaner
 
-def debian():
```

Comment:
This removes support for ALL of Debian, oops!

We just want to remove Debian 10 "buster" and not the URLs with bullseye (11) and bookworm (12).